tools/_setup_generation/step_viselements.py

```diff
@@ -86,8 +88,9 @@ def load_elements(self, elements_json_path: str, prefix: str, doc_pages_path: st
                                 default_property = property[__class__.NAME]
                             del property[__class__.DEFAULT_PROPERTY]
                 element_desc[__class__.DEFAULT_PROPERTY] = default_property
+
             # Resolve inheritance
-            def merge(element_desc, parent_element_desc, default_property) -> Optional[str]:
+            def merge(element_desc, parent_element_desc, default_property, FL_et, FL_pet) -> Optional[str]:
```

Comment:
FL & pet ?

Reply:
Pfff. That was for debugging purpose. Thanks!!